code/src/components/AddTaskForm.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from '../reducers/tasks'
+
+import './addtaskform.css'
+
+export const AddTaskForm = () => {
+    const [name, setName] = useState("")
```

Comment:
maybe this variable should be called something else than name? maybe text or todo or task woulf be a better name?